src/modules/mavlink/mavlink_mission.cpp

```diff
@@ -981,6 +981,10 @@ MavlinkMissionManager::parse_mavlink_mission_item(const mavlink_mission_item_t *
 		case MAV_CMD_DO_CHANGE_SPEED:
 		case MAV_CMD_DO_SET_SERVO:
 		case MAV_CMD_DO_DIGICAM_CONTROL:
+		case MAV_CMD_IMAGE_START_CAPTURE:
```

Comment:
This is really picky, but can you keep the entire thing sorted by value? It makes it easier to keep it consistent in all the different places.